package.json

```diff
@@ -55,6 +55,7 @@
     "@emotion/styled": "^11.10.5",
     "@floating-ui/react-dom": "^1.0.1",
     "@leeoniya/ufuzzy": "^1.0.8",
+    "@wordpress/core-data": "^6.29.0",
```

Comment:
@fabiankaegy I've installed this so that I can import the `Taxonomy` type. Can you confirm if this is the correct way to go?

Reply:
@Sidsector9 I would move it to a `devDependency` because the package won't actually get bundled. But yeah this makes sense to me :)